semanticcpg/src/main/scala/io/shiftleft/semanticcpg/passes/cfgdominator/CfgDominator.scala

```diff
@@ -1,6 +1,8 @@
 package io.shiftleft.semanticcpg.passes.cfgdominator
 
-class CfgDominator[Node](adapter: CfgAdapter[Node]) {
+import io.shiftleft.codepropertygraph.generated.nodes.StoredNode
+
+class CfgDominator[Node <: StoredNode](adapter: CfgAdapter[Node]) {
```

Comment:
Having the CfgDominator class completely independent of our node representation was a deliberate decision. Why do you want to enforce unnecessary type restrictions here?